src/art.rs

```diff
@@ -386,11 +411,12 @@ impl<P: PrefixTrait + Clone, V: Clone> Node<P, V> {
     }
 
     #[inline]
-    pub fn get_value(&self) -> Option<(&V, &u64)> {
+    pub fn get_value(&self) -> Option<(&P, &V, &u64)> {
         let NodeType::Leaf(leaf) = &self.node_type else {
             return None;
         };
-        Some((&leaf.value, &leaf.ts))
+        // TODO: should return copy of value or reference?
```

Comment:
I absolutely dont know, but I think reference is better since it is tied with the lifetime of the object anyway? If that isn't sufficient then the copy can be used to fix it.